microcosm_eventsource/event_types.py

```diff
@@ -195,6 +203,52 @@ def all_states(cls):
                     next_states.append(new_state)
                     known_states.add(new_state)
 
+    @classmethod
+    def all_states_and_events(cls):
```

Comment:
sadly, it's easier to have 2 different functions for `all_transitions` and `all_states_and_events`